libs/langchain/tests/integration_tests/vectorstores/test_cratedb.py

```diff
@@ -247,6 +308,7 @@ def test_cratedb_with_filter_match() -> None:
     assert output in [
         [(Document(page_content="foo", metadata={"page": "0"}), 2.1307645)],
         [(Document(page_content="foo", metadata={"page": "0"}), 2.3150668)],
+        [(Document(page_content="foo", metadata={"page": "0"}), 2.4458315)],
```

Comment:
@matriv: That's just been a fluke again. ;]

Reply:
That is a good idea. I've submitted a corresponding patch. Thank you.

- GH-21